src/languages/en.js

```diff
@@ -149,6 +149,8 @@ export default {
         someone: 'Someone',
         total: 'Total',
         edit: 'Edit',
+        showMore: 'Show more',
+        showLess: 'Show less',
```

Comment:
I see that "Show less" copy is added already but not used yet. Is this feature a follow-up, not in this PR?